chat-server/server.js

```diff
@@ -14,13 +22,44 @@ const welcomeMessage = {
 
 //This array is our "data store".
 //We will start with one message in the array.
-//Note: messages will be lost when Glitch restarts our server.
 const messages = [welcomeMessage];
 
-app.get("/", function (request, response) {
+app.get("/", (request, response) => {
   response.sendFile(__dirname + "/index.html");
 });
 
-app.listen(process.env.PORT,() => {
+app.get("/messages", (request, response) => {
+  response.json(messages);
+});
+app.get("/messages/:ID", (request, response) => {
+  const oneMess = request.params.ID;
+  console.log(oneMess);
+  const filterdMessage = messages.find((one) => {
+    console.log(one.id);
+    return one.id == oneMess;
+  });
+  response.json(filterdMessage);
+});
+app.post("/messages", (req, res) => {
+  const bod = req.body;
+  const newMessage = {
+    id: messages.length,
```

Comment:
We ~always want IDs to be unique, so that no two messages have the same ID (otherwise your `GET /messages/:ID` endpoint won't be able to get some messages). How sure are you that that `messages.length` will always give you a unique value that no other message has?